lib/views/staging-view.js

```diff
@@ -138,14 +139,22 @@ export default class StagingView {
       conflicts: this.props.mergeConflicts || [],
       staged: this.props.stagedChanges,
     });
+    const currentlySelectedItems = this.selection.getSelectedItems();
 
     if (this.props.resolutionProgress !== oldProps.resolutionProgress) {
       await this.resolutionProgressObserver.setActiveModel(this.props.resolutionProgress);
     }
 
-    const selectionChanged = !isEqual(previouslySelectedItems, this.selection.getSelectedItems());
-    if (selectionChanged) {
-      this.debouncedDidChangeSelectedItem();
+    if (this.activeFilePatch) {
+      this.quietlySelectItem(this.activeFilePatch.getFilePath(), this.activeFilePatch.getStagingStatus());
+    } else {
+      const isRepoSame = oldProps.workingDirectoryPath === this.props.workingDirectoryPath;
+      const selectionsPresent = previouslySelectedItems.size > 0 && currentlySelectedItems > 0;
```

Comment:
🚨 It looks like `currentlySelectedItems` is an Array. Did you mean `currentlySelectedItems.length > 0`?

Also: the test suite is ✅ even though this is here. Does that mean we're missing a test case, or does it mean that the `currentlySelectedItems` isn't actually needed?